lms/templates/discussion/_filter_dropdown.html

```diff
@@ -34,19 +35,19 @@
     </li>
 </%def>
 
-<div class="forum-nav-browse-menu-wrapper" style="display: none">
+<div class="forum-nav-browse-menu-wrapper" style="display: none" role="combobox" aria-label="${_("Discussion thread list")}" aria-expanded="true" aria-owns="discussion_topics_listbox">
     <form class="forum-nav-browse-filter">
         <label>
             <span class="sr">${_("Filter Topics")}</span>
-            <input type="text" class="forum-nav-browse-filter-input" placeholder="${_("filter topics")}">
+            <input type="text" aria-autocomplete="list" class="forum-nav-browse-filter-input" placeholder="${_("filter topics")}">
             <span class="icon fa fa-filter" aria-hidden="true"></span>
         </label>
     </form>
-    <ul class="forum-nav-browse-menu">
-        <li class="forum-nav-browse-menu-item forum-nav-browse-menu-all">
+    <ul class="forum-nav-browse-menu" role="listbox" id="discussion_topics_listbox">
+        <li class="forum-nav-browse-menu-item forum-nav-browse-menu-all" role="option">
             <button type="button" class="forum-nav-browse-title">${_("All Discussions")}</button>
```

Comment:
We actually don't want these 'options' to be buttons, because buttons are focusable. Plain text is fine.  HOwever, you have to add javascript listeners to them that performs the same function the buttons currently to.  Each list item should also have a unique ID that is referenced by an aria-activedescendent attribute on  the input type=text above (role=combobox).  HItting the updown arrow keys when on the input should highlight the items in the list.